neural_modelling/src/neuron/c_main.c

```diff
@@ -15,13 +15,13 @@
  *
  */
 
-#include "../common/in_spikes.h"
+#include <in_spikes.h>
 #include "neuron.h"
```

Comment:
Surely we should be consistent here and use `#include <…>` for all of these?

Reply:
The difference is that <in_spikes.h> is brought in by an -I ... while "neuron.h" is local to c_main.c

May be possible/ cleaner to do all as <...> but that is also out of scope of this PR